### DISCUSSION TITLE
Question regarding approved file formats for Orthophoto atom feed

### DISCUSSION THREAD

**Discussion by esband** [INSPIRE-MIF/helpdesk-validator#986] [Q&A] [answered]
This is not a change proposal, rather a question regarding the technical guidelines:

We have problems to validate our service Orthophoto Download, Inspire. The service is an atom feed with the latest nationwide coverage of orthophotos in colour with a resolution of 0,5 metres. 
The downloaded data consists of a zipped file containing one orthophoto (covering 5x5 km) in LZW compressed GeoTIFF, two coordinate files (.tfw and .tab) and three metadata files in GeoJSON (containing metadata for the orthophoto, the aerial photos it consists of and the delivery).
The validator test doesn’t start at all, which is strange. Must the file format be GML, are Geotiff and GeoJSON not approved? 
How must we change our service so that it validates correctly?
Grateful for some advice!

**Comment by fabiovinci** [comment]
Dear @esband,

thank you for your question. Since it seems more related to the Validator, I will move this issue to the [dedicated repository](https://github.com/INSPIRE-MIF/helpdesk-validator/issues).

In any case, the ATOM feed can contain a link to a zip file, so the problem should be related to others aspects.

Could you please provide your ATOM service to reproduce the issue?

**Comment by esband** [comment]
The problem is that the service, at least right now, is not open and free of charge, so providing the link to the service will not make it possible to download any files. The main concern we have is if the format must be gml, instead of the JSON we deliver today?

**Comment by fabiovinci** [comment, marked as answer]
Dear @esband,

there is no specific check about the format.
In any case, if in your metadata you declare that the dataset is conformant to IR 1089, you should provide the related gml file, but this is not tested automatically.
I hope this helps you.

**Comment by fabiovinci** [comment]
Dear @esband,

I will convert this issue into a discussion to leave it open for further input.
